lib/process-services-cloud/src/lib/process/process-filters/components/edit-process-filter-cloud.component.ts

```diff
@@ -514,7 +519,21 @@ export class EditProcessFilterCloudComponent implements OnInit, OnChanges, OnDes
         return (
             this.processFilterCloudService.isDefaultFilter(this.processFilter.name) &&
             this.actionDisabledForDefault.includes(action.actionType)
-        );
+        ) ? true : this.hasFormChanged(action);
+    }
+
+    hasFormChanged(action: ProcessFilterAction): boolean {
+        if (action.actionType === EditProcessFilterCloudComponent.ACTION_SAVE) {
```

Comment:
Can we refactor the function to make it simpler? I suggest this:
`return action.actionType === EditProcessFilterCloudComponent.ACTION_SAVE || action.actionType === EditProcessFilterCloudComponent.ACTION_SAVE_AS ? !this.formHasBeenChanged : false`